src/promptflow-azure/pyproject.toml

```diff
@@ -36,6 +36,12 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "<4.0,>=3.8"
+azure-core = ">=1.26.4,<2.0.0"
```

Comment:
This should depends on promptflow-devkit?

Reply:
OK

Reply:
yes.